pipeline_dp/aggregate_params.py

```diff
@@ -198,7 +198,7 @@ def __post_init__(self):
 
         if self.public_partitions:
             raise ValueError(
-                " AggregateParams.public_partitions is deprecated. Please use public_partitions argument in DPEngine.aggregate insead."
+                "SumParams.public_partitions is deprecated. Please read core API for each backend use case instead"
```

Comment:
@dvadym, Is this correct? Do we want users to pass `public_partitions` as arguments to the private backends, which in the end will be calling the `dp_engine.aggregate()` function?

Reply:
Probably it's better to write about anonymous Sum transform (it's present on both Beam and Spark APIs), i.e.
... is deprecated. Please read API documentation for anonymous Sum transform. 

The same for other transforms